libraries/AP_OpticalFlow/OpticalFlow.cpp

```diff
@@ -52,8 +52,8 @@ const AP_Param::GroupInfo OpticalFlow::var_info[] = {
     // @DisplayName: Flow sensor yaw alignment
     // @Description: Specifies the number of centi-degrees that the flow sensor is yawed relative to the vehicle. A sensor with its X-axis pointing to the right of the vehicle X axis has a positive yaw angle.
     // @Units: cdeg
-    // @Range: -18000 +18000
-    // @Increment: 1
+    // @Range: -17999 +18000
```

Comment:
why is it we use one 17999 limit on these? it's AP_Int16 afaik.

Reply:
aside from that fact that 180 and -180 would be identical values with the same range, I dont know...its the pattern though